chapter_01/hinglish/introduction.ipynb

```diff
@@ -0,0 +1,1346 @@
+{
```

Comment:
<p>Machine doesn't understand our language. It only understands the binary language(combination of 0 and 1). But for humans, it'd be impractical to write in this format. To solve this problem, programming languages were created. Programming language allows us to write in the format that we understand.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/abhiabhi94/learn-python/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>yes, i think you're right, technically, the original text seems to me like pointing out the reason for creating machines. The issue with our suggestion is it introduces a new term binary. Any suggestions for possible workaround it?</p>

Reply:
<p>It only understands <s>the binary language(combination of 0 and 1)</s>  only 0's and 1's. </p>